SWIPs/swip-honey_to_money.md

```diff
@@ -9,29 +9,29 @@ Created: 31-07-2019
+Nodes keep track of the balances with each other nodes using an internal reference unit: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
 
 * The Swarm developers plan to initially set the price of honey very low and progressively increase this price to the point where it is sufficiently high to incentivize nodes to offer their service to the network while being low enough not to scare away potential users of Swarm. 
 * We expect the price of the counterpart currency to fluctuate in value against the costs of operating a Swarm node. As the real benefits (i.e. the worth of your compensation) of operating a Swarm node shouldn't fluctuate, the honey price should fluctuate inversely against the fluctuations of currency. If this happens, the real benefit stays equal, regardless of price fluctuations of the currency in which compensation is paid. 
 * The costs of operating a Swarm node is expected to decrease over time, as hardware becomes cheaper, the code more optimized etc. The price of honey is therefore expected to decrease to stay competitive to other storage solutions. 
 
-This update process should be atomic: either all nodes upgrade or none. If we would update the prices in the Swarm source code (as is currently the case), it will affect that, while not all nodes are on the same release, nodes with By externalising the price to an oracle, we can completely decouple the basic functioning of Swarm - which is heavily focused on data availability, delivery, and security considerations - from the price discovery. By building support for price oracles, we can coordinate price changes across the network, and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. This SWIP makes no recommendation for the eventual pricing mechanism, it only prepares the network for adopting them in future.
+This update process should be atomic: either all nodes upgrade or none. If we would update the prices by releasing a new version of the Swarm source code (as is currently the case), this would be violated, as long as not all nodes are on the same release. By externalising the price to an oracle, we can completely decouple Swarm's core functions from price discovery. By building support for price oracles, we can coordinate price changes across the network, and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. This SWIP makes no recommendation for the eventual pricing mechanism, it only prepares the network for adopting them in future.
```

Comment:
Be consistent with UK vs American English, e.g. "externalising" vs "incentivization"

Reply:
Good comment. What is the generally accepted spelling in Swarm?

Reply:
there is **no problem whatsoever** in using -ise in one word and -ize in the next. I happily mix and match my spelling :D

Reply:
Is that your personal view @homotopycolimit ? Or is it some internationally accepted view that it can happily be mixed and matched. I personally don't like that but if it is common I don't mind.